internal/handlers/publishers.go

```diff
@@ -52,7 +52,7 @@ func (h *PublisherHandler) CreatePublisherHandler(w http.ResponseWriter, r *http
 		// Se o repositório retornar o erro de que o publisher já existe
 		//  retorna 409 Conflict.
 		if errors.Is(err, repository.ErrPublisherAlreadyExists) {
-			errorMessage := fmt.Sprintf("Erro: A editora %q já está cadastrada.", name)
+			errorMessage := fmt.Sprintf(`Erro: A editora %q já está cadastrada.`, name)
```

Comment:
```suggestion
			errorMessage := fmt.Sprintf("Erro: A editora %q já está cadastrada.", name)
```

Esse arquivo não precisa ser alterado, a correção foi feita no teste.